tests/kafkatest/services/security/security_config.py

```diff
@@ -172,6 +173,52 @@ def kafka_opts(self):
         else:
             return ""
 
+    def set_acls(self, protocol, kafka, zk, topic, group):
```

Comment:
This new acl logic looks completely disjoint from the state in `SecurityConfig` objects. Would it make sense to break this out into a separate `KafkaAcl` class in this same directory?